src/MolSim.cpp

```diff
@@ -12,6 +12,7 @@
 #include "spdlog/async.h"
 #include "spdlog/sinks/stdout_color_sinks.h"
 #include "spdlog/spdlog.h"
+#include "io/input/SimulationParams.h"
 
 int main(int argc, char* argsv[]) {
     auto [input_file_path, output_dir_path, delta_t, end_time, fps, video_length, log_level] = parse_arguments(argc, argsv);
```

Comment:
parse_arguments needs to be rewritten to use the new Simulationparams object. 

Both params from the file and the cli need to be merged somehow.

+ I think if two conflicting params are supplied, the CLI should be prefered since it was explicitely requested that way